docs/src/userguide/language_basics.rst

```diff
@@ -44,72 +47,137 @@ the use of ‘early binding’ programming techniques.
 C variable and type definitions
 ===============================
 
-The :keyword:`cdef` statement is used to declare C variables, either local or
-module-level::
+C variables can be declared by using the :keyword:`cdef` statement, by annotating
+variable by special cython type or using function ``declare()``. Statement :keyword:`cdef`
+and function ``declare()`` can declare either local or
+module-level variables, but annotated version currently supports only local variables:
 
-    cdef int i, j, k
-    cdef float f, g[42], *h
+.. tabs::
 
-and C :keyword:`struct`, :keyword:`union` or :keyword:`enum` types:
+    .. group-tab:: Pure Python
 
-.. literalinclude:: ../../examples/userguide/language_basics/struct_union_enum.pyx
+        .. code-block:: python
 
-See also :ref:`struct-union-enum-styles`
+            global_x = declare(cython.int)
+            def main():
```

Comment:
Why global_x only added to pure version and content of main function must be module-level

Reply:
`global_x` was added just to show how to declare global variables (using `declare()` function).

Reply:
You could use declare in both versions or use cdef in cython version instead of declare, i think I'll look better